docs/docusaurus/versioned_docs/version-1.0.0/orc8r/deploy_terraform.md

```diff
@@ -44,6 +44,8 @@ keypair that you imported or created in the above step:
 $ cat vars.tfvars
 db_password = "foobar"
 nms_db_password = "foobar"
+nms_using_mariadb = true
```

Comment:
Is this temporary ? Why do we need to do this for NMS alone ?

Reply:
This is temporary, and should be removed in 1.6. Only NMS is using MariaDB, and the terraform files are set up to name the MariaDB instance as "nms", so I think the name specifying NMS is natural here.

Reply:
If we are unifying this we might as well get rid of these differences ? Is the idea that in case we break something, this might give us a work around ?

Reply:
Both need to be deployed at some point to migrate the data through sequelize

Reply:
I don't feel that we have to pass this additional complexity during upgrade time and having to require user to later turn off the NMS DB. I feel it might be better to have a migration script prior to upgrade, enable us from moving NMS db tables to postgres.